testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -325,12 +325,14 @@ def params_from_base_suite_setup(request):
 
     suite_source_db = None
     suite_db = None
+    suite_db_log_files = None
     if create_db_per_suite:
         if enable_file_logging and liteserv_version >= "2.5.0":
             cbllog = FileLogging(base_url)
             cbllog.configure(log_level="verbose", max_rotate_count=2,
-                             max_size=1000 * 512, plain_text=True)
-            log_info("Log files available at - {}".format(cbllog.get_directory()))
+                             max_size=1000000 * 512, plain_text=True)
```

Comment:
it might effect log rotation tests if size of the log file increased , number of rotation files will reduce and tests might fail. If that is not the case, can you make sure log rotation tests looks okay

Reply:
log rotation has there own config and doesn't depend on the logs create over here.